inc/functions.php

```diff
@@ -1808,7 +1808,18 @@ function is_moderator($fid=0, $action="", $uid=0)
 
 	$user_perms = user_permissions($uid);
 
-	$plugins->run_hooks("is_moderator");
+	$hook_args = array(
+		'fid' => $fid,
+		'action' => $action,
+		'uid' => $uid,
+		'user_perms' => &$user_perms',
+	);
+
+	$plugins->run_hooks("is_moderator", $hook_args);
+	
+	if (isset($hook_args['is_moderator'])) {
+		return true;
```

Comment:
Oh, my suggestion was (is) that rather than `return true;` this line be `return $hook_args['is_moderator'];`, so that the hook can force a false return too.

Reply:
I think that is a good idea.

Reply:
```suggestion
		return $hook_args['is_moderator'];
```